package.json

```diff
@@ -90,8 +90,8 @@
     "react-test-renderer": "^15.5.4",
     "rimraf": "^2.6.1",
     "shelljs": "^0.7.7",
-    "stylelint": "^7.12.0",
-    "stylelint-config-sass-guidelines": "^2.2.0",
+    "stylelint": "~8.0.0",
```

Comment:
stylelint documented their new [semantic version policy](README.md#semantic-versioning-policy). They now recommend the use of the tilde (`~`) in `package.json`, e.g. `"stylelint": "~8.0.0"`, to guarantee the results of your builds ([#1865](https://github.com/stylelint/stylelint/issues/1865)).